Spanner/tests/Unit/BackupTest.php

```diff
@@ -75,6 +76,7 @@ public function setUp()
         $this->lroCallables = [];
         $this->expireTime = new \DateTime("+ 7 hours");
         $this->createTime = $this->expireTime;
+        $this->versionTime = new \DateTime("- 2 hours");
```

Comment:
I don't think this will work, because the database needs to be 2 hours old for the backup to succeed. In the other libraries, we are setting this to be `now()` or the database `earliestVersionTime`.

Reply:
This is a unit test, not an integration test. I'll add an integration test to match the other libraries :+1:

Reply:
Thanks!